server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -250,6 +252,10 @@ impl Upsert for RequisitionRow {
             Ok(Some(self.clone()))
         )
     }
+
+    fn as_mut_any(&mut self) -> Option<&mut dyn Any> {
```

Comment:
this is needed to downcast to requisition row from Boxed upsert in requisition translation test